test/index.cpp

```diff
@@ -1591,3 +1591,17 @@ TEST(Index, Issue1867ChainedIndexingLeak)
                       &lock_bytes, &lock_buffers);
     ASSERT_EQ(0u, lock_buffers);
 }
+
+TEST(Index, InvalidSequence0)
```

Comment:
InvalidSequence0,1,2 are bad names. Perhaps should be InvalidNegativeStep would be more informative.

Reply:
Not sure where you are getting with this. Next test case also has negative step, should we add another word to differentiate from the first one ? That way, we will have lengthy names and you might say not to that again.

Reply:
The idea is that if a test fails we need to be able to figure out why it failed without looking at the code. If the name is descriptive enough we can see a pattern and not have to go digging in the test to figure out what is going on. Long names are fine